packages/devtools_app/lib/src/screens/app_size/app_size_controller.dart

```diff
@@ -48,12 +50,23 @@ class AppSizeController {
   /// The node set as the analysis tab root.
   ///
   /// Used to build the treemap and the tree table for the analysis tab.
-  ValueListenable<TreemapNode?> get analysisRoot => _analysisRoot;
-  final _analysisRoot = ValueNotifier<TreemapNode?>(null);
+  final analysisRoot =
```

Comment:
Let's switch this back so that the `ValueNotifier` is still a private variable (begins with an underscore) and the `ValueListenable` is public (no underscore)

Reply:
SelectionNotifier takes a ValueNotifier so will be keeping analysisRoot as a ValueNotifier